test/components/Auth/test-auth.jsx

```diff
@@ -4,7 +4,7 @@ import { shallow } from 'enzyme';
 
 import Auth from 'components/Auth/Auth.jsx';
 
-export default describe('Auth.jsx', () => {
+describe('Auth.jsx', () => {
```

Comment:
As it turns out the export is not strictly necessary, so I decided to remove it. There shouldn't be a problem leaving the others in, I just felt that this was a bit cleaner.